src/requests/hafas/journey_match.ts

```diff
@@ -0,0 +1,22 @@
+import {getBaseUrl} from "../../server/base-url";
+
+export type Station = {
+  name: string
+}
+
+export type Trip = {
+  id: string
+  destination: Station,
+  origin: Station,
+  line: {
+    name: string
+  }
+}
+
+export default async function journeyMatch(input: string, date: Date): Promise<Array<Trip>> {
+  const response = await fetch(`${getBaseUrl()}/api/hafas/journey_match?q=${input}&date=${date}`, {
```

Comment:
getBaseUrl() is server side code, which won't work on the client

Reply:
It does work and @StckOverflw's code does it as well